Source/WebCore/css/html.css

```diff
@@ -55,6 +55,10 @@ address, article, aside, div, footer, header, hgroup, main, nav, section {
     display: block;
 }
 
+audio:not([controls]) {
```

Comment:
please move this down to be in the `/* media elements */` "section", ideally right below the existing `audio` ruleset

Reply:
Done